src/dso_api/dynamic_api/temporal.py

```diff
@@ -166,13 +224,23 @@ def url_parameters(self) -> dict[str, str]:
             return {}
 
 
+def get_request_date(request: Request) -> datetime:
+    """Find the temporal query date, associated with the request object."""
+    try:
+        # Make sure all queries use the exact same now, not seconds/milliseconds apart.
+        return request._now
+    except AttributeError:
+        request._now = now()
```

Comment:
If we do want to keep this, I think it might be cleaner to set it from some piece of middleware. Maybe the AuthMiddleware could be expanded to become an `AdornMiddleware` that sets various bits of context info on the request.

Reply:
Not crucial though, feel free to ignore.